src/main/java/analysis/bachelorarbeit/FilterTripsCSV2.java

```diff
@@ -0,0 +1,258 @@
+package analysis.bachelorarbeit;
```

Comment:
Your personal Bachelor Thesis files don't belong into this repository. Please remove those from the history of this branch. We can do that together if you don't want to do it yourself. 

Otherwise have a look here: https://stackoverflow.com/a/52643437 and the answer below that one.